ERCS/erc-7579.md

```diff
@@ -224,11 +234,25 @@ interface IModule {
     function onInstall(bytes calldata data) external;
     function onUninstall(bytes calldata data) external;
     function isModuleType(uint256 typeID) external view returns(bool);
+    function getModuleTypes() external view returns (uint256[]);
 }
 ```
 
 Modules MUST revert if `onInstall` or `onUninstall` was unsuccessful.
 
+
+##### Module Management and Lifecycle
+
+**Module State Management:**
+
+- On installation: Smart accounts MUST initialize module states as specified.
+- On uninstallation: Smart accounts SHOULD clearly state whether the module's state is preserved or reset, per module documentation.
+
+**Mandatory Validation Module:**
```

Comment:
how is this different to https://github.com/zeroknots/ERC-minimalisticMSA/blob/bf1f94f03d2c1ba92670f6f7fe8805a6877c52f1/ERCS/erc-7579.md?plain=1#L223-L226

Reply:
I didn't notice the duplication. Removed